fedimint-build/src/lib.rs

```diff
@@ -66,7 +66,7 @@ fn set_code_version_inner() -> Result<(), String> {
         // state by replacing the middle with 0s. This should
```

Comment:
I'm a bit confused by this comment. Are we supposed to keep the hash 40 hex chars long? If so, wouldn't we want to skip the 8 chars of "0" in the middle? i.e. `[(40 - 16 + 8)..40]`?

Reply:
Ahh scratch that.